dessia_common/core.py

```diff
@@ -379,6 +379,8 @@ def load_from_stream(cls, stream: dcf.JsonFile):
 
         Should be consistent with save_to_stream method.
         """
+        warnings.warn("This method is deprecated and will be removed in a future version."
+                      " Please use the `from_json_stream` method instead.", DeprecationWarning)
         dict_ = json.loads(stream.read().decode('utf-8'))
```

Comment:
In order to properly implement backward compatibility, you should call the new function here since they are the same.